charts/ccd-api-gateway-web/values.yaml

```diff
@@ -9,7 +9,7 @@ nodejs:
     CORS_ORIGIN_METHODS: GET,POST,OPTIONS,PUT,DELETE
     IDAM_SERVICE_NAME: ccd_gw
```

Comment:
do we need to bump up chart version?

Reply:
I assume this will happen at the next scheduled release if this is the develop branch?